db/migrate/20160415121724_create_cloud_services.rb

```diff
@@ -0,0 +1,21 @@
+class CreateCloudServices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cloud_services do |t|
+      t.string :ems_ref
+      t.string :source
+      t.string :binary
+      t.string :host_name
+      t.string :status
+      t.string :scheduling_status
+      t.string :zone_name
```

Comment:
This is availability_zone as defined in OpenStack - Compute Service carries only name of the zone - maybe better name would be availability_zonename? It is used to match CloudService with AvailabilityZone.